trident/line_database.py

```diff
@@ -49,7 +50,7 @@ class Line:
         Example: 'I' for neutral species, 'II' for singly ionized, etc.
 
     :wavelength: float
-    
+
```

Comment:
I accidentally did a "Trim Trailing Whitespace" on saving, which brought all these irrelevant diffs :( Sorry about that.